tsconfig.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
A future suggestion if more repos for entropic use TypeScript. Consider creating a master tsconfig.json and inherit from it. For an example, see https://github.com/sindresorhus/refined-github/blob/master/tsconfig.json

Reply:
Well, yes, of course but the project is reasonably small now so I'm not sure why we'd do that? As far as I can tell, this is the first `tsconfig.json` in any entropic project. Naturally, we'll extend a base config if more appear.

Reply:
For sure, that's why I prefaced it with `A future suggestion...` 😉

Reply:
While I appreciate the suggestion, I suggest we avoid getting off topic and stick to reviewing the PR according to its scope. 💞